js/filter-panel.js

```diff
@@ -25,21 +27,24 @@ function FilterPanel(options) {
   this.$toggle.on('click', this.toggle.bind(this));
 
   this.filterSet = new FilterSet(this.$form).activate();
-  if (!_.isEmpty(this.filterSet.serialize())) {
-    this.show();
-  }
 
   this.adjust();
 }
 
 FilterPanel.prototype.adjust = function() {
-  if ($('body').width() > 768) {
+  if ($(document).width() > 768) {
     this.show();
   } else if (!this.isOpen) {
     this.hide();
   }
 };
 
+FilterPanel.prototype.setHeight = function() {
+  if ( $(document).width() > 768 && this.$dataContainer.height() > this.$body.height() ) {
```

Comment:
Oh yep, it could be. We don't use it much, but every once in a while. Should we move the `GLOBALS` object to helpers.js in fec-style?